src/Spryker/Glue/AppMerchantBackendApi/Dependency/Facade/AppMerchantBackendApiToTranslatorFacadeBridge.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Glue\AppMerchantBackendApi\Dependency\Facade;
+
+class AppMerchantBackendApiToTranslatorFacadeBridge implements AppMerchantBackendApiToTranslatorFacadeInterface
+{
+    /**
+     * @var \Spryker\Zed\Translator\Business\TranslatorFacadeInterface
+     */
+    protected $translatorFacade;
+
+    /**
+     * @param \Spryker\Zed\Translator\Business\TranslatorFacadeInterface $translatorFacade
+     */
+    public function __construct($translatorFacade)
+    {
+        $this->translatorFacade = $translatorFacade;
+    }
```

Comment:
```suggestion

    public function __construct(protected TranslatorFacadeInterface $translatorFacade)
    {
    }
```

Reply:
We have this sniff, I've decided not to exclude it here.
https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Internal/SprykerBridgeSniff.php#L79

Reply:
Sure thing sorry, I missed that its a bridge...